### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by ethantkoenig**
#228 

image

image

### CODE REVIEW

File: templates/home.tmpl [hunk 1]
```diff
@@ -21,21 +16,30 @@
...
-			<p>DCS allows you to explore the work of various translation projects for various languages by different users. To work on your own project or collaborate on an existing project, you first need to <a href="/user/sign_up">register</a>. This account can both be used here and on <a href="https://unfoldingword.org/translationstudio" target="_blank">translationStudio</a>.</p>
-			<p>To learn how to translate, check, publish, and use our tools, check out <a href="https://unfoldingword.org/academy" target="_blank">translationAcademy</a>.</p>
+			<h3><i class="octicon octicon-person"></i> Get Started</h3>
+			<p>First <a href="https://git.door43.org/user/sign_up" rel="nofollow">register</a> for a Door43 account. This account can both be used here on DCS and in apps that connect to DCS.</p>
+			<p>If you are a translator, we suggest you start working in <a href="https://unfoldingword.org/translationstudio" rel="nofollow">translationStudio</a>,  <a href="http://www.autographa.com/" rel="nofollow">Autographa</a>, or  <a href="https://unfoldingword.org/tc/" rel="nofollow">translationCore</a>.</p>
```
Comment (jag3773) [inline templates/home.tmpl]: Please point the tS link at https://unfoldingword.org/ts/ .

File: templates/home.tmpl [hunk 2]
```diff
@@ -21,21 +16,30 @@
...
 		</div>
 
 		<div class="eight wide left column">
-			<h3>Communication</h3>
+			<h3><i class="octicon octicon-comment-discussion"></i> Communication</h3>
 			<p>If you have questions, you can post them on the <a href="https://team43.slack.com/messages/helpdesk" target="_new">#helpdesk</a> channel in Slack (see below) or email <a href="mailto:help@door43.org">help@door43.org</a>.
-			<p>The DCS community uses Slack to communicate. When you <a href="https://door43.org/en/slack" target="_blank">join our team on Slack</a>, you can collaborate with other translators, share your ideas with the developers, and find help for your translation, checking, publishing, and technology needs. Slack works well in a Browser or you may download a native client for it as well.
+			<p>The Door43 community uses Slack to communicate. When you <a href="https://door43.org/en/slack" rel="nofollow">join Team43 on Slack</a>, you can collaborate with other translators, share your ideas, and find help for your translation, checking, publishing, and technology needs.</p>
```
Comment (jag3773) [inline templates/home.tmpl]: Please point the Slack link to https://door43.org/en/contact/ .

**Comment by ethantkoenig** [comment]
@jag3773 Fixed. As a heads up, the old links came from [the wiki](https://git.door43.org/Door43/pages/wiki/Welcome-to-Door43-Content-Service%21), so maybe we also want to change the links there?

**Comment by jag3773** [comment]
Instead of maintaining the separate Wiki page, can we have a persistent link to this page your creating?  If so, then I'd like to replace the link in the header for "About DCS" to point at that internal page.  Does that make sense?

**Comment by richmahn** [comment]
The reason we didn't have a link to the home page before is because the Gogs (Gitea) router has the Home page only available for those who are not signed in. This would either require a change to the router (which I didn't make to avoid conflicts with upstream should anything change) or create another page like the home page, but that would mean two would have to be maintained still....

**Comment by jag3773** [comment]
I'll merge this for now and create a new issue for the About page.

**Pull request merged**